lib/test.php

```diff
@@ -0,0 +1,8 @@
+<?php
+require_once('crt.php');
+
+$crt = new CRT();
+$crt->init();
+var_dump($crt->error_name(0));
+$crt->clean_up();
+?>
```

Comment:
Its a good practice in PHP not to close last ?> to avoid output of trailing space.